src/main/resources/anniv5Resources/localization/eng/UIstrings.json

```diff
@@ -105,7 +105,8 @@
       "StS Modding Community",
       "Adds a new character: The Packmaster.",
       "ALL PACKS MODE. Ignores Custom Draft",
-      "One Frame For All. All Pack cards have normal Packmaster frames (Requires Restart)."
+      "One Frame For All. All Pack cards have normal Packmaster frames (Requires Restart).",
+      "Share Content. Some Relics/ Potions can appear on other characters (Requires Restart)."
```

Comment:
Happy with where we landed with this -- people can opt in if they want. Also good to see we call out that it requires a restart.